src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -107,12 +107,14 @@
 	private Boolean enabled;
 	private Boolean seeded;
 
-	private final boolean use_cdm;
 	private final String cdm_project_prefix;
 
+    private KibanaSeed kibanaSeed;
```

Comment:
I converted from a Static utility class to an instance and it is used to seed dashboards.  Reformatted the class.